spec/factories/report.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :v2_report, class: 'V2::Report' do
+    before(:create) do |instance, context|
```

Comment:
Does it have to be in the `before` bloc? Can't this be moved to the `to_create`?